forms-html/yazen_lawin/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+
+<body>
+<div style="font-weight: bold;">
+
+
+
+<form>
+    <label for="email"> Enter your email:</label> <br>
+    <input type="email" id="email" name="email" value="osama@recoded.com" required > <br>
+    <label for="username">user name:</label><br>
+    <input type="text" id="username" name="username"  value="osama" required ><br>
+    <label for="password">password:</label><br>
+    <input type="password" id="password" name="password"  value="12458" required ><br>
+    <label for="quantity">quantity(between 1 and 5):</label>
+    <input type="Number" id="quantity" name="quantity" min="1" max="5" required ><br>
+    <label for="volume">Volume(between 0 and 50):</label>
+    <input type="range" id="volume" name="volume" min="0" max="50" required ><br>
```

Comment:
always put new line between each label input pair like this :

```
    <label for="email"> Enter your email:</label> <br>
    <input type="email" id="email" name="email" value="osama@recoded.com" required > <br>

    <label for="username">user name:</label><br>
    <input type="text" id="username" name="username"  value="osama" required ><br>

    <label for="password">password:</label><br>
    <input type="password" id="password" name="password"  value="12458" required ><br>

    <label for="quantity">quantity(between 1 and 5):</label>
    <input type="Number" id="quantity" name="quantity" min="1" max="5" required ><br>

    <label for="volume">Volume(between 0 and 50):</label>
    <input type="range" id="volume" name="volume" min="0" max="50" required ><br>
```
you code will be more readable and more clean